modules/apps_api/README.md

```diff
@@ -0,0 +1,22 @@
+# Apps Api
+
+This project uses MIT-LICENSE.
```

Comment:
Does it? https://github.com/department-of-veterans-affairs/vets-api/blob/master/LICENSE.md

Reply:
This is an error that has been fixed in the latest commit.